lib/rpmsg/rpmsg_virtio.c

```diff
@@ -371,6 +371,7 @@ static void *rpmsg_virtio_get_tx_payload_buffer(struct rpmsg_device *rdev,
 {
 	struct rpmsg_virtio_device *rvdev;
 	struct rpmsg_hdr *rp_hdr;
+	uint8_t virtio_status;
```

Comment:
why not use status

Reply:
`virtio_get_status()` takes a pointer to a 8bit value, using status which is an `int` could case endianness issues. I could have turned status into a `uint8_t`, but it is used as a status return and there would be a narrowing. Since they are used for different things in different ways I made a new variable.

Reply:
ok